index.js

```diff
@@ -93,16 +93,26 @@
         (report.warningCount && options.stopOnWarning);
     }
 
+    function reportErrorTally() {
+      if(totalErrorCount > 0) {
+        log.warn(chalk.green('There are a total of ') +
+        chalk.white.bgRed(totalErrorCount) + chalk.green(' errors'));
+      }
+    }
+
     return function(content, file, done) {
       var report = cli.executeOnFiles([file.path]);
 
       log.debug('Processing "%s".', file.originalPath);
       if(shouldStop(report)) {
+        reportErrorTally();
```

Comment:
It looks to be reporting regardless of the state of this if/else if/else branch, so perhaps it should just do that one:

```js
log.debug('Processing "%s".', file.originalPath);

reportErrorTally();

if(shouldStop(report)) {
  //....
```